ZukoAzula2016/src/org/team1540/zukoazula/ChallengeBrake.java

```diff
@@ -12,14 +12,15 @@
     private static final BooleanInput secondTrigger = ZukoAzula.controlBinding.addBoolean("Challenge Brake B");
     private static final BooleanInput composedTriggers = FRC.inTeleopMode().and(firstTrigger).and(secondTrigger);
 
-    private static final FloatOutput servos = FRC.servo(0, 0, 1).combine(FRC.servo(1, 1, 0));
+    // needs to be this for Zuko; the reverse for Azula.
     private static final BooleanCell servoState = new BooleanCell();
     public static final BooleanInput driveBrake = servoState;
 
     public static void setup() {
         servoState.setTrueWhen(composedTriggers.onPress());
         servoState.setFalseWhen(FRC.startDisabled);
```

Comment:
Relating to the subject, should we deploy/retract the brake when we enter pit mode?